tests/structures/test_datamodel.py

```diff
@@ -576,7 +572,7 @@ def __rmul__(self, other):
             print(3 * p1)
         """, run_in_function=False)
 
-    @expected_failing_versions(['3.4'])
+    @expectedFailure
```

Comment:
What's the underlying cause of this new (or, at least, more expanded) failure?

Reply:
This does look like an expansion, but it is actually a tightening. The wrapping class of this set of tests was all annotated as expectedFailure, but for some reason 3.4 wasn't respecting that (I added those 3.4 only failures to each test in this class earlier). I've removed the expectedFailure from the class now and applied it to just this one method. It does seem like an issue that 3.4 didn't respect the class being marked as expectedFailure, but hopefully Dan cuts 3.4 out of the picture shortly.